src/Task5.java

```diff
@@ -27,22 +27,34 @@ public static void main(String[] args) {
                 System.out.println("Такого дня нету");
         }
         //Задача1
-        int clientOs = 0;
+        int clientOs = 0; // clientOs, где 0 - Ios, 1 - Android.
         if (clientOs == 0) {
             System.out.println("Вы пользуетесь Ios. Установите версию приложения для Ios по ссылке");
         } else if (clientOs == 1) {
             System.out.println("Вы используете Android. Скачайте версию приложения для Android");
         }
         // Задача 2.
+        // Ios - 0, Android - 1.
+        int clientOs1 = 0;
         int clientDeviceYear = 2015;
+        String operationSystem;
+        if (clientOs1 == 0) {
+            operationSystem = "Ios";
+        } else {
+            operationSystem = "Android";
+        }
+        String message;
         if (clientDeviceYear < 2015) {
-            System.out.println("Установите облегченую версию приложения Ios" + clientOs + " по ссылке");
+            message = "Установите облегченую версию для " + operationSystem + " по ссылке";
         } else {
-            System.out.println("Установите облегченную версию приложения для Android " + clientOs + " по ссылке");
+            message = "Установите облегченную версию приложения для " + operationSystem + " по ссылке";
         }
+        System.out.println(message);
+
+
         //Задача 3.
         int year = 2021;
-        if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
+        if ((year < 1584 && year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
```

Comment:
```suggestion
        if ((year > 1584 && year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
```
по заданию если БОЛЬШЕ